<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>492</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>492</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgb(46, 32, 58);
border-top-right-radius:25px ;
border-bottom-left-radius:25px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pg_login">
          <widget class="QLineEdit" name="login_user">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>100</y>
             <width>191</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">/*background-color: rgb(57, 46, 69); */
border: 1px solid rgb(57, 46, 69);
border-radius: 12px;
color :#B48FCB;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="login_passwd">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>140</y>
             <width>191</width>
             <height>31</height>
            </rect>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">/*background-color: rgb(57, 46, 69); */
border: 1px solid rgb(57, 46, 69);
border-radius: 12px;
color: #B48FCB;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_login">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>210</y>
             <width>311</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(46, 32, 58);
border:none;
}
QPushButton:hover{
	background-color: rgb(57, 46, 69);
}
QPushButton:pressed{
	background-color: rgb(112, 0, 132);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/cil-user.png</normaloff>icons/cil-user.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>140</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#b590cc;&quot;&gt;Password :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_to_create">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>180</y>
             <width>141</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color : #B48FCB;</string>
           </property>
           <property name="text">
            <string>create an account</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>100</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#b590cc;&quot;&gt;Username :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>180</y>
             <width>161</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#b590cc;&quot;&gt;you dont have an account ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_exit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(46, 32, 58);
border:none;
border-radius:  12px;
}
QPushButton:hover{
	background-color: rgb(187, 33, 88);
}

QPushButton:pressed{
	background-color: rgb(227, 4, 4);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/x.png</normaloff>icons/x.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
          <widget class="QLabel" name="invalid">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>40</y>
             <width>311</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'SFMono-Regular','Consolas','Liberation Mono','Menlo','monospace'; font-size:16pt; color:#f9000e;&quot;&gt;Invalid&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pg_create">
          <widget class="QPushButton" name="btn_creat">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>210</y>
             <width>331</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(46, 32, 58);
border:none;
}
QPushButton:hover{
	background-color: rgb(57, 46, 69);
}
QPushButton:pressed{
	background-color: rgb(112, 0, 132);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/cil-user-follow.png</normaloff>icons/cil-user-follow.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>90</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#b590cc;&quot;&gt;Username :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="creat_user">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>90</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">/*background-color: rgb(57, 46, 69); */
border: 1px solid rgb(57, 46, 69);
border-radius: 12px;
color :#B48FCB;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="creat_passwd1">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>130</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">/*background-color: rgb(57, 46, 69); */
border: 1px solid rgb(57, 46, 69);
border-radius: 12px;
color :#B48FCB;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>130</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#b590cc;&quot;&gt;Password :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="creat_passwd2">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>170</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">/*background-color: rgb(57, 46, 69); */
border: 1px solid rgb(57, 46, 69);
border-radius: 12px;
color :#B48FCB;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>170</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#b590cc;&quot;&gt;Confirm pass:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="go_to_login">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>260</y>
             <width>141</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color : #B48FCB;</string>
           </property>
           <property name="text">
            <string>Go to login Page</string>
           </property>
          </widget>
          <widget class="QLabel" name="invalid1">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>20</y>
             <width>311</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'SFMono-Regular','Consolas','Liberation Mono','Menlo','monospace'; font-size:16pt; color:#f9000e;&quot;&gt;Invalid&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
